cmd/boulder-wfe2/main.go

```diff
@@ -295,9 +305,15 @@ func main() {
 	}
 
 	var noncePrefixKey string
```

Comment:
Let's make this a []byte. That means we'll need to change the type of key in nonce.DerivePrefix, and of rncKey in wfe.NewWebFrontEndImpl and on the WebFrontEndImpl struct.